eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -34,18 +33,17 @@ steps:
   
 - pwsh: |
     git clone https://github.com/${{ parameters.TargetDocRepoOwner }}/${{ parameters.TargetDocRepoName }} ${{ parameters.WorkingDirectory }}/repo
-
+ 
     try {
       Push-Location ${{ parameters.WorkingDirectory }}/repo
-
-      Write-Host "git checkout ${{ parameters.SourceBranchName }}"
```

Comment:
Why do we not need this checkout anymore? If it isn't needed then we can also remove the try/finally block and the directory changing.